README.md

```diff
@@ -31,16 +31,19 @@ It should be possible to build everything without Qt Creator, but the guide assu
    Client executables will appear in the `exec` folder, and the server executable will appear in the `griefly-server` folder.  
    Client executables depend from various dlls, so it is needed to manually place them to the `exec` folder.
 
-**Note:** It is supposed to perform build from an active git repository (`git describe --tags --abbrev=4` will be called during the process).
+**Note:** It's supposed to be built from an active git repository (`git describe --tags --abbrev=4` will be called during the process).
 
 How to build everything on Linux
 --------------------------------
 
-1. Install dependencies. Look into `.travis.yml` file for clues.
-2. `./make.sh`. Built project will be placed under `exec` directory. Server will be
+1. Install dependencies. Look into `.travis.yml` file for clues. 
+2. `cd` into installation directory.
+3. `git clone https://github.com/griefly/griefly.git`
+4. `cd griefly`
+4. `./make.sh`. Built project will be placed under `exec` directory. Server will be
    built in `gopath/src/griefly-server` directory.
 
-**Note:** It is supposed to perform build from an active git repository (`git describe --tags --abbrev=4` will be called during the process).  
+**Note:** It's supposed to be built from an active git repository (`git describe --tags --abbrev=4` will be called during the process) -- if you recieve a `fatal: Not a git repository` error, then `git init` will surpress the issue, but will interfere with network connections.
```

Comment:
1. Two spaces at the end of the line should not be removed - it is a part of the markdown language. Without it looks wrong (you can check it yourself).
2.  `git init` should not be mentioned at all. It is a bad way to hide the issue and does not give any profit. The error means only one thing - the project should be cloned from a git repo.